### PR TITLE
Update Japanese translation for 6.13.0

### PR DESCRIPTION
**Pull request by kakakaya**
- Update Japanese translation for AutoMuteUs 6.13.0
    - Including changes around `DisplayRoomCode` settings
- Improve translation
- Remove `locales/translate.*.toml`
    - According to [LOCALIZATION.md](https://github.com/denverquane/automuteus/blob/master/LOCALIZATION.md), it should be removed always and should be ignored by `.gitignore` as I think.

### CODE REVIEW

File: locales/active.ja.toml [hunk 1]
```diff
@@ -907,8 +919,8 @@ hash = "sha1-33216b66ff7cd33ff9c82ece012055bdefd291ba"
 other = "<true/false (有効/無効)>"
 
 ["settings.AllSettings.MuteSpectators.desc"]
-hash = "sha1-6b5c5f2f2ef1b8c1326aa751aaaf5f885012c0ee"
-other = "観客を死んだプレイヤーと同じように扱うべきかどうか (音声ルールに従わせます)。\n**注: セルフホストしていない、または Premium worker ボットを使用している場合、遅延や速度低下が発生する可能性があります！**"
+hash = "sha1-e8a9a8d8000ed326525f1548be0cca7b6a9402d4"
+other = "観客を死んだプレイヤーと同様に扱うべきかどうか (音声ルールに従わせます)。\n**注: セルフホストしていない場合、または Premium worker ボットを使用している場合、遅延や速度低下が発生する可能性があります！**"
```
Comment (kurokobo) [inline locales/active.ja.toml:923]: > セルフホストしていない場合、または Premium worker ボットを使用している場合

should be "セルフホストしていない場合、または Premium worker ボットを使用していない場合".

There is a mistake in the original translation, where the word "not" in the original refers to both "self-hosting" and "using".

> when not self-hosting, or using a Premium worker bot!

File: locales/active.ja.toml [hunk 2]
```diff
@@ -1167,7 +1191,7 @@ hash = "sha1-9c659ccece4b459f6c91514f2191d49d6181b95b"
 other = "観客を死んだプレイヤーと同じようにミュートしなくなります"
 
 ["settings.SettingMuteSpectators.true_noMuteSpectators"]
-hash = "sha1-cb62081e4c9e37177e32ad916c63ef5ff4a812fe"
+hash = "sha1-fdc47919bc2805420aa085dace4eab2d815d5c75"
 other = "観客を死んだプレイヤーと同じようにミュートします。\n\n**注: セルフホストしていない、または Premium worker ボットを使用している場合、遅延や速度低下が発生する可能性があります！**"
```
Comment (kurokobo) [inline locales/active.ja.toml:1195]: Same as the first one.

**Comment by kurokobo** [comment]
Thanks for sending PR 😃 
Added some comments.

File: locales/active.ja.toml (hunk 1 above)
Comment (kakakaya) [reply locales/active.ja.toml]: Thanks for your review! Your indication is correct and original translation was wrong. But, the new translation also has ambiguous expression.

>  セルフホスト**していない場合、または** Premium worker ボットを使用 **していない場合**、遅延や速度低下が発生する可能性があります

This sentence can be  misread as follows:, "セルフホストを**しており、かつ** Premium worker ボットも使用している**のではなければ**、遅延や速度低下が発生する可能性があります", by De Morgan's laws.

The original sentence was "when not self-hosting, or using a Premium worker bot!", and this means "not self-hosting" and "not using a Premium worker bot!".

----

Then, I new translation: "セルフホストをしておらず、 Premium worker ボットも使用していない場合". It is hard to misread and more natural. Committed in 3dc25f0.

**Comment by kurokobo** [comment]
Thanks, LGTM about improvement of Japanese translation 😃 
It is ultimately up to denverquane to decide whether or not to delete `translation.*.toml`.

**Comment by kurokobo** [comment]
@kakakaya 
I am unfortunate that this PR will not be merged, but starting with version 7, AutoMuteUs uses Crowdin for localization and translation: <https://automuteus.crowdin.com/automuteus>
Some part of your translations are ported to Crowdin, but if you want to improve current translation, please feel free to make suggestion or upvoting on Crowdin. Thanks 😃

**Pull request closed without merging**